src/Jackett.Common/Definitions/anilibria.yml

```
@@ -54,6 +62,13 @@ search:
       optional: true
     title:
       text: "{{ .Result.title_ru }} / {{ .Result.title_en }}{{ if .Result.title_alternative }} / {{ .Result.title_alternative }}{{ else }}{{ end }} ({{ .Result.year }}) [{{ .Result._quality }}]{{ if .Result._series }} - {{ .Result._series }}{{ else }}{{ end }}"
+      filters:
+        - name: re_replace
+          args: ["(\\([А-Яа-яЁё\\W]+\\))|(^[А-Яа-яЁё\\W\\d]+\\/ )|([а-яА-ЯЁё \\-]+,+)|([а-яА-ЯЁё]+)", "{{ if .Config.striprussian }}{{ else }}$1$2$3$4{{ end }}"]
+        - name: re_replace
+          args: ["[\\:\\-\\/\\|\\[\\]]", " "]
+        - name: append
+          args: "{{ if .Config.addrussian }} - RUSSIAN{{ else }}{{ end }}"
```

```suggestion
          args: "{{ if .Config.addrussian }} RUS {{ else }}{{ end }}"
```
it is usually only necessary to specify RUS to flag the language, and avoiding a dash prefix helps reduce the likelyhood of -groupname clashes.

---

Still is the same issue tho, no?

some.title.year.quality-group => some.title.year.quality-groupRUS

---

there is a space in front of the RUS, does this not make a difference?

---

In the case of this indexer, we're building the title and there's no release group.

As for any similar future issue:

- `title-Group - RUS`, `title-Group RUS` and  `title-Group (RUS)` return `Group` as the release group
- `title-Group [RUS]` returns `RUS` as the release group
- `title - RUS` returns no release group.